dbaas/maintenance/tasks.py

```diff
@@ -110,3 +112,84 @@ def execute_scheduled_maintenance(self, maintenance_id):
                                    details='Maintenance executed succesfully')
 
     LOG.info("Maintenance: {} has FINISHED".format(maintenance,))
+
+
+def region_migration_prepare(infra):
+    instance = infra.instances.first()
+    ChangeTTLTo5Minutes(instance).do()
```

Comment:
Theses classes are used in workflow execution, it's a inherited of Step, so, we can only pass 'instance' type objects. 
The implementation of theses classes are doing it, "converting the time"